### ISSUE TITLE
Fails install without '.' in @INC

### ISSUE THREAD

**Issue by kentfredric** [houseabsolute/perl-code-tidyall#74]
On 5.25.10 with -Ddefault_inc_excludes_dot
```
Configuring Code-TidyAll-0.57 ... Can't locate inc/Util.pm in @INC (you may need to install the inc::Util module) (@INC contains: /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/site_perl/5.25.10 /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10/x86_64-linux /home/kent/perl5/perlbrew/perls/5.25.10-nossp-sdbm-nopmc-nodot/lib/5.25.10) at Makefile.PL line 146.
BEGIN failed--compilation aborted at Makefile.PL line 146.
Configuring Code-TidyAll failed.
```

**Comment by karenetheridge** [comment]
I would suggest naming the test package from `inc::Util` to just `Util`, and adding `use lib 'inc';` -- that's a bit safer than `use lib '.';`

**Comment by autarch** [comment]
This is fixed in c7c6ce7e2a515bed1cecd090b6cdba4ff50ce422

**Comment by autarch** [comment]
Fixed in 0.58
